tiatoolbox/utils/transforms.py

```diff
@@ -164,6 +164,25 @@ def bounds2size(bounds, origin="upper"):
             Defaults to upper.
     """
     left, top, right, bottom = bounds
+    origin = origin.lower()
     if origin == "upper":
         return np.array([right - left, bottom - top])
-    return np.array([right - left, top - bottom])
+    if origin == "lower":
```

Comment:
I couldn't get this one. What do you mean by origin="lower"? and why would we need top-bottom?

Reply:
This is if the original point is inverted e.g. matplotlib uses lower left by default. This defaults to upper as it is the convention for graphics contexts and images.